routes/api/v1/forecasts.js

```diff
@@ -0,0 +1,122 @@
+var express = require('express')
+var router = express.Router()
+
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('../../../knexfile')[environment]
+const database = require('knex')(configuration)
+const fetch = require('node-fetch')
+
+function findByKey (key) {
+  return database.select('id').from('users').where('api_key', key).first()
```

Comment:
you can move this into a User model function since youll prob use it in other routes

Reply:
I ended up moving that function because I did have to use it again later. Thanks for the input!